docs/requirements.txt

```diff
@@ -5,4 +5,12 @@
 sphinx==4.5.0
 sphinx_rtd_theme==1.0.0
 readthedocs-sphinx-search==0.1.2
-myst-parser==0.18.0
```

Comment:
why are you pinning to a fixed version?

Reply:
No specific reason. I will remove them then.